buildmaster/LHCB_DY_13TEV_DIELECTRON/metadata.yaml

```diff
@@ -0,0 +1,53 @@
+# Previously called `LHCB_Z_13TEV_DIELECTRON`
+# author: "Tanjona R. Rabemananjara"
+setname: "LHCB_DY_13TEV_DIELECTRON"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1607.06495"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1477581"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1477581"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "LHCb"
+
+implemented_observables:
+- observable_name: "y"
+  observable:
+    description: "Differential cross-section of Z-->ee as a function of Z-rapidity"
+    label: r"$d\isgma / d|y|$"
+    units: "[fb]"
+  process_type: "DY"
```

Comment:
Since this is a reimplementation, please use the same `process_type` that it had before.
For [this dataset/observable](https://github.com/NNPDF/nnpdf/blob/master/nnpdfcpp/data/commondata/DATA_LHCB_Z_13TEV_DIELECTRON.dat) is `EWK_RAP`.

(the process types are [here](https://github.com/NNPDF/nnpdf/blob/94917a4bdddc5e99f1835d61e7a0540f8ef07052/validphys2/src/validphys/filters.py#L19))